### DISCUSSION TITLE
I accidentally switched to Japanese

### DISCUSSION THREAD

**Discussion by RoundaboutSnake** [WerWolv/ImHex#1254] [Feedback]
NVM

If someone accidentally switches languages, then what is the fail-safe to roll it back? Uninstall and reinstall didn't help. TY

**Comment by paxcut** [comment]
open imhex and on the next to the last menu select the fifth entry (the first one in second block). that should open the settings. On the settings c;lick the second tab and when that opens pull down the third menu. on it select second entry for english

**Comment by paxcut** [comment]
Also edit the json for settings and choose any language. That is located in the config folder in the imhex folders (in user/AppData/local/imhex/config by default ). If you delete that file then all the settings will be reset,

**Comment by WerWolv** [comment]
For future record, it's possible now to reset back all settings to their default by running ImHex through the command line with the following flag:
```bash
imhex --reset-settings
```
